packages/google_mlkit_document_scanner/ios/Classes/GoogleMlkitDocumentScannerPlugin.h

```diff
@@ -0,0 +1,4 @@
+#import <Flutter/Flutter.h>
```

Comment:
Name of this file should be: 
 `GoogleMlKitDocumentScannerPlugin.h`

Reply:
okay, i thought i changed this.